docs/components/console/manage-organization/external-sso.md

```diff
@@ -34,3 +34,7 @@ For Azure AD, you will need to provide:
 - The client secret value
 
 To generate the client on your end, you will need to use the Camunda **Redirect URL** `https://weblogin.cloud.camunda.io/login/callback `. Ensure you attach the user permissions `Users > User.Read`.
+
+## Troubleshooting
```

Comment:
Would you consider this troubleshooting? Did this topic come up based on feedback where a user couldn't configure properly?